apps/ingest-croissant/requirements.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/aperture-data/aperturedb-python@croissant_data_enrichment
```

Comment:
this dependency with a specific branch should be removed before merging

Reply:
yes, will create a SDK release today before merging